NEWS

```diff
@@ -11,6 +11,44 @@
 ### Features:
 ### Bugfixes:
 
+## 1.16.0 (January 21, 2024)
+### Features:
+#### UCP
+* Multiple performance improvements
+* Multiple improvements in error and debug messages
+#### UCT
+* Improved intra-node collectives performance for small messages
+* Use NIM/MEMIC for SHMEM atomics in CX7
+#### RDMA CORE (IB, ROCE, etc.)
+* Added FLID based routing support
+* Added latency and min_zcopy configuration variables to ROCm-IPC
+* Optimized xpmem tag matching
```

Comment:
tag matching happens in UCP for XPMEM, right ?

Reply:
removed this line (seems it was done for 1.15)